src/skin/qml/qmlvisibleeffectsmodel.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+#include <QAbstractListModel>
+#include <memory>
+
+#include "effects/effectsmanager.h"
+
+namespace mixxx {
+namespace skin {
+namespace qml {
+
+class QmlVisibleEffectsModel : public QAbstractListModel {
+    Q_OBJECT
+  public:
+    enum Roles {
```

Comment:
It works, but it's kind of clunky because of all the casting between int and the enum value. Since the enum part of the class body and each member has a "Role" suffix, I think it's okay like this.

@uklotzde opinions?